input/fsh/SD_TumorSize.fsh

```diff
@@ -119,7 +119,8 @@ Description: "Represents a tumor after it has been removed from the body. Prior
 * identifier contains tumorIdentifier 0..*
 * identifier[tumorIdentifier] only BodyStructureIdentifier
 * identifier[tumorIdentifier] ^short = "Identifier to associate this specimen with a specific Tumor"
-* identifier[tumorIdentifier] ^definition = "To associate this with a specific BodyStructure conforming to the Tumor profile, add an identifier with a value that matches a persistent identifier from `BodyStructure.identifier.value` that is unique in the context of the Patient."
+// FHIR-32352
+// * identifier[tumorIdentifier] ^definition = "To associate this with a specific BodyStructure conforming to the Tumor profile, add an identifier with a value that matches a persistent identifier from `BodyStructure.identifier.value` that is unique in the context of the Patient."
```

Comment:
You should be updating and improving the definition, rather than eliminating it (according to FHIR-32352)

Reply:
One of the criticisms was using the definition for instructions on usage.   This was moved to the usage of Tumor Specimen -- https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/laterality/StructureDefinition-mcode-tumor-specimen.html